File: barcosEnContienda/src/gestorAplicacion/Lugares/Isla.java
package gestorAplicacion.Lugares;

import java.util.ArrayList;

import gestorAplicacion.Embarcaciones.Barco;
import gestorAplicacion.Items.Botin;
import gestorAplicacion.Items.Objeto;
import gestorAplicacion.Personas.Tripulacion;

public class Isla extends Lugar {
//Nuevos atributos
	public String nombre;
	public Tripulacion tripulacion;
	public Botin botin;
	public ArrayList<Objeto> objetosAcceso;
	public Barco barco;
	
	public Isla(String nombre, Tripulacion tripulacion, Botin botin, ArrayList<Objeto> objetosAcceso) {
		super(nombre, tripulacion, botin, objetosAcceso);
	}
//Nuevo metodo
	public void vincularTripulacion(Tripulacion tripulacion){
		this.tripulacion = tripulacion; 
	}
}

File: practica-2-grupo-1-equipo-7/src/gestorAplicacion/lugares/isla.py
from gestorAplicacion.lugares.lugar import Lugar
class Isla(Lugar):
    
    def __init__(self, nombre, tripulacion, botin, objetoacceso, barco=None):
        
        super().__init__(nombre, tripulacion, botin, objetoacceso)
    
    def vincularTripulacion(self,tripulacion):
        self._tripulacion = tripulacion
